response-to-reviewers.tex

```diff
@@ -560,12 +576,10 @@ \subsection*{Random thoughts I had while reading the paper, to discuss (or
 0, 50, 0.0001\}
 \end{point}
 \begin{reply}
-TODO
-
-\ggcomment{
-Say something about sparse versus dense.
-Then refer to our docs so that we don't have to copy/paste the docs in the manuscript.
-}
+YAML supports a similar type of shorthand, and there are various
+examples in the tutorial where we use it to increase legibility,
+e.g.
+\url{https://popsim-consortium.github.io/demes-spec-docs/main/tutorial.html#exponential-size-changes}
```

Comment:
It seems you've interpreted Laurent's point here to refer to flow-style versus block-style YAML? I interpreted it more about using a matrix `M` to represent migrations (a dense representation) versus what we do where a source/dest migration list element refers to `M[source][dest]` (a sparse representation).

Reply:
Yes, that's true. He sort of implied both I think, so I went with the version that's practical?

Feel free to update in a follow up, I don't have  strong opinions here.